WDAC-Policy-Wizard/app/src/Helper.cs

```diff
@@ -1301,6 +1312,30 @@ public void AddNewSeparationLine(string subTitle)
             }
         }
 
+        /// <summary>
+        /// Removes any path of the form :\Users\john.doe\xyz with %USERPROFILE%
+        /// </summary>
+        /// <param name="msg">Sanitized message</param>
+        /// <returns></returns>
+        private string SanitizeMsg(string msg)
+        {
+            string unsanitizedPath = @":\Users\";
+            if (!msg.Contains(unsanitizedPath))
+            {
+                return msg; 
+            }
+
+            string sanitizedMsg;
+            string userProfileString = "%USERPROFILE%";
+            int startOfUnsantizedMsg = msg.IndexOf(unsanitizedPath) + unsanitizedPath.Length;
+            int endOfUnsanitizedMsg = msg.IndexOf("\\", startOfUnsantizedMsg );
```

Comment:
You'll have a runtime error with this function if it somehow operates on the directory, 

":\Users\john.doe